code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -96,13 +100,27 @@
 				update_icon()
 		else
 			to_chat(user, "<span class='warning'>Seems like someone already drew something on this helmet's visor!</span>")
-
+		return
+	if(istype(hitting_item, /obj/item/clothing/head))
+		var/obj/item/clothing/hitting_clothing = hitting_item
+		if(hitting_clothing.clothing_flags & PLASMAMAN_HELMET_EXEMPT)
+			to_chat(user, "<span class='notice'>You cannot place [hitting_clothing.name] on the helmet!</span>")
+			return
+		if(attached_hat)
+			to_chat(user, "<span class='notice'>There's already something placed on the helmet!</span>")
+			return
+		attached_hat = hitting_clothing
+		to_chat(user, "<span class='notice'>You placed the [hitting_clothing.name] on the helmet! Keep in mind this is merely cosmetic: hat lights, stats or anything else won't work!</span>")
+		hitting_clothing.forceMove(src)
+		update_icon()
 /obj/item/clothing/head/helmet/space/plasmaman/worn_overlays(isinhands)
 	. = ..()
 	if(!isinhands && smile)
 		var/mutable_appearance/M = mutable_appearance('icons/mob/clothing/head.dmi', smile_state)
 		M.color = smile_color
 		. += M
+	if(!isinhands && attached_hat)
+		. += attached_hat.build_worn_icon(default_layer = HEAD_LAYER, default_icon_file = 'icons/mob/clothing/head.dmi')
 	if(!isinhands && !up)
 		. += mutable_appearance('icons/mob/clothing/head.dmi', visor_icon)
 	else
```

Comment:
I just tested and the crayon smile shows up above the hat
![image](https://user-images.githubusercontent.com/42877449/144326737-82b0fb3f-572f-482e-8625-7099cff75d8d.png)
I tried pretty hard to find a solution, but nothing I tried worked. it should be fine though, since it will probably be rare that some simultaneously 1. drew a smile on their helmet 2. is wearing a hat on their helmet, and 3. the hat they're wearing goes over their face

Reply:
if it isn't fixed in this pr and it's a problem, someone who knows how to can fix it later

Reply:
I thought I put exempt on this one. The idea is to only allow them to wear hats, but not things that can cover the whole face. Gonna check on it.

Reply:
i was thinking the rad helmet was supposed to be, but hadn't said it yet.

> not things that can cover the whole face

hmmm
![image](https://user-images.githubusercontent.com/42877449/144330467-d0f1ad15-591a-4db6-80d3-8ab4b2478cda.png)

Reply:
Yeah I left the sans one because it's a costume and it was funny, but well, there seems to be a problem between crayon and hats so I'll add exempts to all the stuff that covers the whole helmet.

Reply:
honestly, you can leave the sans one as a joke. i think that'd be funny